src/website/src/pages/ec/components/tag/docs/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  Tags are used to provide contextual information; find related content using
+  keywords, labels or links. Tags are also used as keywords in search boxes,
+  filters and category lists. In some cases they can be added or removed by the
+  user.
+</Paragraph>
+
+## Anatomy
+
+There's 2 variant of tags to accommodate different needs
+
+### Link tags
+
+| Name     | Mandatory | Restrictions |
+| -------- | --------- | ------------ |
+| Label    | X         |              |
+| URL-link | X         |              |
+
+### Removable tags
+
+| Name         | Mandatory | Restrictions |
+| ------------ | --------- | ------------ |
+| Label        | X         |              |
+| Dismiss icon | X         |              |
+
+## When to use it
+
+### Link tags
+
+- To navigate to another page/content
+- Use tags to link current content to other related content
+- showing information which helps a user make sense of the content on the page, such as filter parameters or object metadata.
+- Use to display information that clarifies the content on the page, such as metadata.
```

Comment:
in this kind of cases, the tag may not be clickable. So a link is not the best structure here.
Maybe we need to have `<span>` tags...